frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/common/webTerminal.ts

```diff
@@ -10,29 +10,29 @@ import { checkTerminalIcon } from '@console/dev-console/integration-tests/suppor
 import { webTerminalPage } from '@console/webterminal-plugin/integration-tests/support/step-definitions/pages/web-terminal/webTerminal-page';
 
 const idp = Cypress.env('BRIDGE_HTPASSWD_IDP') || 'consoledeveloper';
-const username = Cypress.env('BRIDGE_HTPASSWD_USERNAME') || 'consoledeveloper';
+const username = Cypress.env('BRIDGE_HTPASSWD_BASIC_USERNAME') || 'consoledeveloper';
```

Comment:
Is this really required? Our other e2e tests uses the environment variable `BRIDGE_HTPASSWD_USERNAME` as well. I think we should avoid special cases for different plugins when possible.

Reply:
The main problem after getting the WebTerminal userstories -  we need to use testcases under different roles (basic user and developer user). And when we set up a cluster with several roles - we need to manage login under kubeadmin and, for example, the basic user. Let's look at how we can manage it now. If we look at the https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/plugins/index.js#L66-L70 We have:

 |ENV_VAR|MANAGE ROLE
|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------
|config.env.BRIDGE_HTPASSWD_PASSWORD = process.env.BRIDGE_HTPASSWD_PASSWORD; |we can manage admin user password|
|config.env.BRIDGE_HTPASSWD_PASSWORD = process.env.BRIDGE_HTPASSWD_PASSWORD; |we can manage basic user password|
|config.env.BRIDGE_HTPASSWD_USERNAME = process.env.BRIDGE_HTPASSWD_USERNAME;|we can manage just 1 username - but we need to manage the admin name, basic user, |

And in this case, from my point of we, we can add something like the:
`config.env.BRIDGE_BASIC_PASSWORD = process.env.BRIDGE_BASIC_PASSWORD;`
`config.env.BRIDGE_BASIC_NAME = process.env.BRIDGE_BASIC_NAME;`
`config.env.BRIDGE_DEV_PASSWORD = process.env.BRIDGE_DEV_PASSWORD;`
`config.env.BRIDGE_DEV_NAME = process.env.BRIDGE_DEV_NAME;`
 `config.env.BRIDGE_HTPASSWD_ADMIN_USERNAME = process.env.BRIDGE_HTPASSWD_USERNAME;`
 But I decided not to add this into `index.js` configuration for avoiding unexpected influents on other tests. I just added this part into webterminal test scope. 
 But how organizing all users the questions is opened as for me